uportal-war/src/main/java/org/jasig/portal/rest/PeopleRESTController.java

```diff
@@ -102,4 +102,18 @@ public ModelAndView getPerson(@PathVariable String username,
         return mv;
     }
 
+    @RequestMapping(value = "/people/me.json", method = RequestMethod.GET)
+    public ModelAndView getMe(HttpServletRequest request, HttpServletResponse response) {
+        final IPerson me = personManager.getPerson(request);
+
+        if ( me == null ) {
+            //If null, this person is not logged in.
+            response.setStatus(401);
+            return null;
+        }
+
+        return new ModelAndView("json")
+                .addObject("person", me.getAttributeMap());
```

Comment:
Looks good.  For consistency with the other API responses in this class, consider changing to
`.addObject("person", new NamedPersonImpl(person.getUsername(), person.getAttributeMap());`
True it only adds an additional username attribute that returns your username which is probably not that necessary and nests the attributes one level deeper, but I am thinking it would be good for all of these methods to return an IPersonAttributes for consistency.